src/System.Text.Json/tests/Serialization/NullableTests.cs

```diff
@@ -0,0 +1,253 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Collections.Immutable;
+using System.Linq;
+using System;
+using Xunit;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public static partial class NullableTests
```

Comment:
I think tests where we have nullable value-types and the `IgnoreNullValues` feature would be useful to add.

Reply:
This area is well covered in -https://github.com/dotnet/corefx/blob/5e102fb5d30af4bc940aa519195c0e3dbd933648/src/System.Text.Json/tests/Serialization/Null.ReadTests.cs, 
https://github.com/dotnet/corefx/blob/5e102fb5d30af4bc940aa519195c0e3dbd933648/src/System.Text.Json/tests/Serialization/Null.WriteTests.cs

Reply:
I see just two tests where IgnoreNullValues is true:
https://github.com/dotnet/corefx/blob/5e102fb5d30af4bc940aa519195c0e3dbd933648/src/System.Text.Json/tests/Serialization/Null.WriteTests.cs#L47
https://github.com/dotnet/corefx/blob/5e102fb5d30af4bc940aa519195c0e3dbd933648/src/System.Text.Json/tests/Serialization/Null.ReadTests.cs#L87

And both of them use the same POCO (`TestClassWithInitializedProperties`) with just an `int` for a nullable value-types. We should add more - like DateTime and user-defined structs that are nullable (to force the interaction between these two features covered).